src/bika/lims/browser/analysisrequest/__init__.py

```diff
@@ -34,9 +34,9 @@
 # this comment twice.
 from .view import AnalysisRequestViewView
 
-from .add2 import AnalysisRequestAddView  # noqa: F401
-from .add2 import AnalysisRequestManageView  # noqa: F401
-from .add2 import ajaxAnalysisRequestAddView  # noqa: F401
```

Comment:
Why is it not necessary to have these lines? Are they no longer reported as "unused import"?